.github/workflows/ci.yml

```diff
@@ -19,6 +19,7 @@ jobs:
       with:
         java-version: adopt@1.11
     - run: npm install jsdom@12.2.0
+    - run: docker-compose -f ./fetch/client/src/test/resources/wiremock/docker-compose.yml up -d
```

Comment:
I am not familiar with this kind of setup. When is the server going to be shut down?

Reply:
I do not think that has to be managed - once CI is done, machine gets shutdown together with any containers it spawned.